src/app/test/mocked-objects.ts

```diff
@@ -151,3 +153,118 @@ export const quayToken: Token = {
     'refreshToken': null,
     'userId': 2,
 };
+
+/**
+ * Near identical to the production 'quay.io/wtsicgp/dockstore-cgpmap/cgpmap-cramOut' tool but with many versions and dates removed
+ */
+export const realTool = {
```

Comment:
Sorry, let's move this into the DB dump, then we can use regular migration on upgrades.

Reply:
This is for the unit tests which don't rely on a db dump.  I could add an integration test if needed, but it seems a little overkill for a docker pull command.

Reply:
Yeah, integration test if necessary to avoid the mock.